tools/release-plugin.sh

```diff
@@ -67,30 +56,67 @@ while getopts "v:abh" opt; do
 done
 shift "$(($OPTIND -1))"
 
-# If there are more than two arguments, bail.
-[[ $# -gt 2 ]] && die "Too many arguments specified! Only provide a project and a version number. Got:$(printf ' "%s"' "$@")"$'\n'"(note all options must come before the project slug)"
+# Parse arguments in associated array of plugins/project => version format.
+if [ $# -eq 0 ]; then
+	usage
+fi
+
+declare -A PROJECTS
+while [[ $# -gt 0 ]]; do
+	if [[ "$1" =~ ^[a-zA-Z\-]+$ ]]; then
+		PLUGIN=$1
+	else
+		die "Script must be passed in [project] <version> format. Got $1"
+	fi
+	
+	if [[ "$2" =~ ^[0-9]+(\.[0-9]+)+(-.*)?$ ]]; then
+		PROJECTS["$PLUGIN"]+=$2
+		SHIFT="2"
+	else
+		PROJECTS["$PLUGIN"]=''
+		SHIFT="1"
+	fi
+	shift "$SHIFT"
+done
 
-# Get the project slug.
-PROJECT=
-SLUG="${1#projects/}" # DWIM
-SLUG="${SLUG%/}" # Sanitize
-if [[ -e "$BASE/projects/$SLUG/composer.json" ]]; then
-	yellow "Project found: $SLUG"
-	PROJECT="$SLUG"
+# If we're releasing Jetpack, we're also releasing mu-wpcom-plugin.
+if [[ "${!PROJECTS[*]}" =~ "jetpack" ]]; then
```

Comment:
I don't like this, it's liable to false-positive if any plugin's name happens to contain the string "jetpack". Why not test directly? (and while we're at it, avoid overwriting a passed version for mu-wpcom-plugin)
```suggestion
if [[ -n "${PROJECTS[jetpack]+ok}" && -z "${PROJECTS[mu-wpcom-plugin]+ok}" ]]; then
```